libraries.json

```diff
@@ -112,6 +120,18 @@
     "github": { "user": "purescript", "repo": "purescript-graphs" }
   },
   {
+    "bowerName": "purescript-halogen"
```

Comment:
Thanks! Already fixed, currently building locally to make sure everything is alright.